cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,48 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    """
+    Updates the metadata's first child tag's text.
+    Does not support tags with complex/object-like values.
+    """
+
+    task_options = {
+        "entity": {"description": "Metadata Entity", "required": True},
+        "tag": {
+            "description": "Targeted tag. The text of the first instance of this tag within the metadata entity will be updated.",
+            "required": True,
+        },
+        "value": {
+            "description": "Desired value to set for the targeted tag's text. This value is namespace-injected.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
```

Comment:
I can imagine a few other options that would make this more versatile (such as the option to target all instances of a tag, or to target a nested tag using xpath). But there's also a strong argument to be made for keeping it simple and creating other tasks for other use cases. Any thoughts, @davidmreed

Reply:
I'm on Team Simple for this one. I prefer to have Metadata ETL tasks that clearly bound/state the operation they are capable of executing. This one is the furthest we've gone so far in the direction of general-purpose MDETL tasks, and I think it's fine, but I'm more comfortable with lots of configurability living in abstract base classes and primarily exposing tasks that do one thing explicitly.